### DISCUSSION TITLE
How to use .tsv file for training dalle?

### DISCUSSION THREAD

**Discussion by PoorGear** [lucidrains/DALLE-pytorch#386] [Q&A] [answered]
Hello, I want to train DALLE with Google's conceptual-12m (https://github.com/google-research-datasets/conceptual-12m),but I can't figure out how to use this .tsv datasets for training.
 Is it enough to simply compress the downloaded file into .gz format?
like this;
python train_dalle.py --wds img,cap --image_text_folder ~/cc12m.tsv.gz

This may be missing the point, but please let me know.

**Comment by rom1504** [comment, marked as answer]
Hi, you first need to download the dataset, here is an example on how to do it https://github.com/rom1504/img2dataset/blob/main/examples/cc12m.md

**Reply by PoorGear** [reply]
rom1504
Thank you for the quick answer!
I'm going to try your img2dataset.

**Comment by rom1504** [comment]
btw I'd suggest starting with something like https://github.com/rom1504/kaggle-fashion-dalle

to get good results with cc12m you need many days of training, for that small dataset, you can get ok results in 1 day
